### DISCUSSION TITLE
de/1-02-neuronal-networks/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#52] [General]
# de/1-02-neuronal-networks/

https://ai.lernos.org/de/1-02-neuronal-networks/

**Comment by ghost** [comment]
Typo im zweiten Satz unter dem Bild:

- **Aktuell:** Die _Verbindingen_ darf man sich...
- **Korrekt:** Die _Verbindungen_ darf man sich...

**Comment by rednaxelaesor** [comment]
Geniales Video - Von der ersten bis zu letzten Sekunde!

**Comment by KathaZ** [comment]
A1/Z1: *K*ünstliche Neuronale Netze
A1/Z2: natürlichen neuronale*n* Netzen*,* wie z.B. in einem Gehirn*,* nachempfunden
A1/Z5: Die Verbind*u*ngen
A1/Z8: Llama-2-7B-Modell
